backend/app/Http/Controllers/Category/CategoryController.php

```diff
@@ -19,6 +19,7 @@ class CategoryController extends Controller
     public function index()
     {
         $categories = Category::all();
+        $this->returnAll($categories);
         return response()->json([
             'data' => $categories
         ], 200);
```

Comment:
I think you returnAll should be return in this case cause thats where you call return response.
So code would probably look ike this:
```suggestion
        $categories = Category::all();
        return $this->returnAll($categories);
```

Reply:
ay yes sir, wala diay na remove 😅